apollo-router/src/plugins/telemetry/fmt_layer.rs

```diff
@@ -727,7 +726,7 @@ connector:
 
                 error!(http.method = "GET", "Hello from test");
 
-                let router_events = event_config.new_router_events();
+                let mut router_events = event_config.new_router_events();
```

Comment:
Everything became explicitly mutable.
Previously it also was mutable but hidden under internal Arc + Mutex